ban/core/models.py

```diff
@@ -108,14 +103,25 @@ class Group(BaseGroup):
     laposte = db.CharField(max_length=10, null=True, unique=True)
     ign = db.CharField(max_length=24, null=True, unique=True)
 
+    municipality = db.ForeignKeyField(Municipality, related_name='groups')
+
+    class Meta:
+        abstract = True
+
     @property
     def tmp_fantoir(self):
         return '#' + re.sub(r'[\W]', '', unidecode(self.name)).upper()
 
+    @property
```

Comment:
Ah, that's why get_fantoir was returning a string…
Why making that a dynamic property?